android/app/build.gradle

```diff
@@ -205,6 +205,8 @@ task copyDownloadableDepsToLibs(type: Copy) {
 configurations.all {
     resolutionStrategy {
         force 'com.android.support:support-v4:26.1.0'
+        // our fork has applied a fix for https://github.com/facebook/react-native/issues/31572 uncomment for testing the demo app
```

Comment:
Might be worth putting this comment in the README or Contributing guide for anyone trying to run the repo?